lib/cisco_node_utils/cmd_ref/vrf.yaml

```diff
@@ -41,6 +41,25 @@ destroy:
   ios_xr:
     set_value: 'no vrf <vrf>'
 
+mhost_ipv4:
```

Comment:
I think that combining v4 and v6 does make sense in this case given that the commands are virtually identical except for the afi.
```
get_value '/^mhost <afi> default-interface (\S+)$/'
set_value: '<state> mhost <afi> default-interface <intf>'
```
```